cli/search/search.go

```diff
@@ -42,6 +46,11 @@ const (
 	GitRepoTT        = "https://github.com/tarantool/tt.git"
 )
 
+const (
+	GitRepoTarantoolToClone = "https://github.com/tarantool/tarantool"
```

Comment:
Why aren't the constants above suitable for git clone?

Reply:
fixed